template/scripts/pre-commit.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+echo "Definition of done checklist.\n"
+
+# Read user input, assign stdin to keyboard
+exec < /dev/tty
+
+while read -p "1. With this commit, is the code complete? (Y/n) " yn; do
+    case $yn in
+        [Yy] ) break;;
+        [Nn] ) echo "${RED}Be sure to address all reamining work priot to submittion.${NC}\n"; break;;
+        * ) break;;
+    esac
+done
```

Comment:
This feels ambiguos to me - complete in what sense? Not all tickets will always have all requirements done as a single commit. It may make more sense to break it up. Wouldn't this change here mean every commit has to have every bit of code that the entire ticket needs? Seems unnecessarily restrictive to me.

Reply:
Actually,... all of these feel like things that should be monitored at the PR level rather than the commit level.

Reply:
Reviewing this personally, you are speaking to the level of burden. 

I do have the script so that users can simply hit the return key and it will accept that as a `y`. But are there too many questions? Should there be only one question asking if the user reviewed the Auro Definition of Done document?